frontend/src/util/ga-util.ts

```diff
@@ -1,31 +1,44 @@
-import ReactGA from 'react-ga';
+import firebase from 'firebase/app';
+import 'firebase/analytics';
 import { AppUser } from 'firebase/auth-util';
 
 export const initialize = (): void => {
-  ReactGA.initialize('UA-134683024-1', {
-    debug: process.env.NODE_ENV !== 'production',
-    gaOptions: {
-      sampleRate: 100,
-    },
-  });
-  ReactGA.pageview(window.location.pathname + window.location.search);
+  firebase.analytics().logEvent('screen-view', { screenName: 'Homepage' });
 };
 
-export const setGAUser = ({ uid }: AppUser): void => ReactGA.set({ userId: uid });
+type EventType =
```

Comment:
This is used for custom event. If I don't define it here and put it in type parameter in line 28, typescript will not be happy.